services/blockchain-indexer/shared/indexer/transactionProcessor/pos/stake.js

```diff
@@ -90,29 +97,51 @@ const decrementStakeTrx = async (stake, trx) => {
 	await stakesTable.decrement(decrementParam, trx);
 };
 
+const updateTotalStake = async (changeAmount, dbTrx) => {
+	const tokenID = await getPosTokenID();
+	const tokenKey = `${KEY_VALUE_TABLE_KEYS.TOTAL_STAKED_PREFIX}_${tokenID}`;
+	const curStakedAmount = BigInt(await keyValueTable.get(tokenKey) || 0);
+	const newStakedAmount = curStakedAmount + changeAmount;
+
+	await keyValueTable.set(tokenKey, newStakedAmount, dbTrx);
+};
+
 // eslint-disable-next-line no-unused-vars
 const applyTransaction = async (blockHeader, tx, dbTrx) => {
 	const stakes = await getStakeIndexingInfo(tx);
+	let totalStakeChange = BigInt(0);
 
 	logger.trace(`Indexing transaction ${tx.id} contained in block at height ${tx.height}.`);
```

Comment:
Just to be consistent
```suggestion
	logger.trace(`Indexing stakes in transaction ${tx.id} contained in block at height ${tx.height}.`);
```

Reply:
Please use `Indexing stakes in`.

Reply:
Applied.